aten/src/ATen/native/cudnn/BatchNorm.cpp

```diff
@@ -88,10 +88,10 @@ std::tuple<Tensor, Tensor, Tensor> cudnn_batch_norm(
     mode = CUDNN_BATCHNORM_PER_ACTIVATION;
   } else {
     mode = CUDNN_BATCHNORM_SPATIAL;
-#if CUDNN_VERSION >= 7003
-    if(training)
-      mode = CUDNN_BATCHNORM_SPATIAL_PERSISTENT;
-#endif
+    // TODO(T31829456): The new CUDNN_BATCHNORM_SPATIAL_PERSISTENT mode was
```

Comment:
can you remove reference to internal task T31829456

Reply:
Sorry didn't see this yesterday, removing